build.gradle

```diff
@@ -1,19 +1,12 @@
-group 'cc.jilt' // we need a domain we own for Maven Central, and jilt.org is scalped
-version '1.1'
-
-buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.4'
-    }
+plugins {
```

Comment:
Any reason this is first? Does `plugins` have to be first in the file?

If not, let's keep `group ` and `version` here at the start - the diff will be smaller this way.

Reply:
Yeah plugins has to be first.
```
> startup failed:
  build file 'jilt\build.gradle': 4: only buildscript {} and other plugins {} script blocks are allowed before plugins {} blocks, no other statements are allowed
```